examples/annotation_import/tiled.ipynb

```diff
@@ -77,13 +77,13 @@
     },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    print("   ")
</pre><p>Let's get rid of this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/883/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>